pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/codestyle/xml/BooleanGetMethodName.xml

```diff
@@ -34,6 +34,16 @@ public class Foo {
         ]]></code>
     </test-code>
 
+    <test-code>
+        <description>Should not match on multiple parameters by default</description>
+        <expected-problems>0</expected-problems>
+        <code><![CDATA[
+public class Foo {
+    public Boolean getEnabled(boolean thisIsNotABean);
```

Comment:
```suggestion
    public Boolean getEnabled(boolean thisIsNotABean) { return false; }
```

I know, the code of the existing test cases won't compile either, but if we create new test cases, we should make sure, we write valid code. It doesn't matter so much for PMD at the moment, as we are much more lenient than a real compiler, but that might not always be the case.

This applies also for the other test cases you added.